deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -1,6 +1,11 @@
+
```

Comment:
why the name change? we already change this twice.

Reply:
Was also confused, but based on Jordan's comment

> Re-generate the crds by using the make generate-crds then change to the correct name and yaml format by using our agreed new controller-gen way.

this is how they are generated. I just ran `make generare-crds` locally and for me the files are also generated with the names that Jordan has in his code.

Reply:
Interestingly, my generated files still contain the line breaks which I like more than the very long lines in Jordan's commit. Hm. @zhangtbj I got `controller-gen` installed via the hack/install-controller-gen.sh script. Do you maybe run a different version of the tool?

Reply:
Yes, I use the official `make generare-crds` command by following our doc.

The version I am using is (On Mac):
```
✘-1 $ ./controller-gen --version
Version: v0.4.1
```

Reply:
Funny detail. I just read [newline discrepancy in CRDs generated by controller-gen #514](https://github.com/kubernetes-sigs/controller-tools/issues/514) three times. They describe it the other way around. There, Mac output has line breaks, but Ubuntu not. You're on Mac while I am on Ubuntu and we observe it vice versa. They suggest to pipe through `yq` to prevent large git diffs, but that is not really nice as well. Honestly, this is one of these kinds of things where I wonder how the hack must the code look that produces such a discrepancy. lol

I think I approve, may the second reviewer decide if we can live with this.

Reply:
@zhangtbj but you are also adding new fields on the CRDS, might be an explanation on why will not hurt?

Reply:
I double check this. Its fine for me to keep the new names, code wise it make sense. I had a different opinion before, but I was on the wrong path.